app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.kt

```diff
@@ -122,6 +123,12 @@ class FahrplanFragment : Fragment(), MenuProvider {
     private var lastSelectedSession: Session? = null
     private var displayDensityScale = 0f
 
+    /**
+     * Cache of the already rendered RoomData
+     * Used to redraw only the rooms that really changed
+     */
+    private val renderedRoomGuidHashByRoomName: MutableMap<String, Int> = mutableMapOf()
```

Comment:
With the current implementation the property should be named: `renderedRoomDataHashCodeByRoomGuid`.